client/ignore.lua

```diff
@@ -4,7 +4,7 @@ CreateThread(function()
 			SetScenarioTypeEnabled(sctyp, false)
 		end
 		for _, scmdl in next, Config.BlacklistedScenarios['SUPPRESSED'] do
-			SetVehicleModelIsSuppressed(GetHashKey(model), true)
+			SetVehicleModelIsSuppressed(GetHashKey(scmdl), true)
```

Comment:
```suggestion
			SetVehicleModelIsSuppressed(joaat(scmdl), true)
```
joaat is faster I believe?